extra/modules/fiftyone-devicedetection/src/main/java/org/prebid/server/hooks/modules/fiftyone/devicedetection/model/boundary/EnrichmentResult.java

```diff
@@ -0,0 +1,14 @@
+package org.prebid.server.hooks.modules.fiftyone.devicedetection.model.boundary;
+
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.Collection;
+
+@Builder
+public record EnrichmentResult<DeviceInfoBox>(
```

Comment:
Having it generic doesn't make much sense

Reply:
this was removed